src/JupyterLibrary/clients/jupyterlab/Notebook.robot

```diff
@@ -51,3 +51,14 @@ Run Current JupyterLab Code Cell
     ...    ${nb.find_element_by_xpath('''div${JLAB XP NB TOOLBAR FRAG}//${run icon}''')}
     Click Element    ${run btn}
     Sleep    0.5s
+
+Get Selected Tab ID
+    [Documentation]    Return the tab ID of the selected notebook tab
+    ${active-nb-tab} =    Get WebElement    xpath:${JL_TABBAR_SELECTED_XPATH}
+    ${tab-id} =    Get Element Attribute    ${active-nb-tab}    id
+    [Return]  ${tab-id}
+
+Select Tab By Name
```

Comment:
Similar, as above re: naming convention, and I don't see where `JL_TABBAR_CONTENT_XPATH` is defined.

As other tab-related things, what happens if there's more than one?

Reply:
> As other tab-related things, what happens if there's more than one?


I changed the keyword to select the tab by filepath instead of filename. So, if there are tabs with same name but different filepath, it still works. What do you think?